### PR TITLE
New KL model + laneless toggle

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: selfdrive/ui/qt/offroad/settings.cc [hunk 1]
```diff
@@ -50,13 +50,17 @@ QWidget * toggles_panel() {
...
+  toggles_list->addWidget(horizontal_line());
   ParamControl *record_toggle = new ParamControl("RecordFront",
                                             "Record and Upload Driver Camera",
                                             "Upload data from the driver facing camera and help improve the driver monitoring algorithm.",
                                             "../assets/offroad/icon_network.png");
-  toggles_list->addWidget(horizontal_line());
   toggles_list->addWidget(record_toggle);
+  toggles_list->addWidget(horizontal_line());
+  toggles_list->addWidget(new ParamControl("EndToEndToggle",
+                                            "Ignore lanelines",
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/settings.cc:61]: can we put something like expiremental or alpha in the title?

File: selfdrive/ui/ui.cc [hunk 2]
```diff
@@ -292,6 +292,7 @@ static void update_params(UIState *s) {
 
   if (frame % (5*UI_FREQ) == 0) {
     read_param(&scene.is_metric, "IsMetric");
+    read_param(&scene.end_to_end, "EndToEndToggle");
```
Comment (adeebshihadeh) [inline selfdrive/ui/ui.cc:295]: read this once on rising edge of ignition

**Comment by grekiki** [comment]
That looks exciting

**Comment by wocsor** [comment]
Interesting that "laneless" gets a toggle but not colorblind accessible UI. Has your design philosophy changed?

**Comment by geohot** [comment]
"Ignore lanelines" -> "Disable use of lanes"?

**Pull request merged**